pds_doi_core/cmd/pds_doi_cmd.py

```diff
@@ -47,6 +48,11 @@ def main():
             o_doi_list = list_action.run()
             print(o_doi_list)
 
+        elif action_type == 'release':
```

Comment:
We can replace this section of if elif with the following piece of code:

       if action_type in {'draft', 'check', 'list', 'release', 'reserve'}:
            module = importlib.import_module(f'pds_doi_core.actions.{action_type}')
            action_class = getattr(module, f'DOICoreAction{action_type.capitalize()}')
            action = action_class()
            action.parse_argument_from_cmd(arguments)
            output = action.run()
            print(output)
        else:
            logger.error(f"Action {action_type} is not supported yet.")

This will require more work on the parse argument method because I am not sure if it is available for all actions. 
Coud you work on that ?

Reply:
This was quite a bit of work as all actions related code has to change to support the import_module function.

However, it does make the main cmd code much cleaner.

Reply:
I did some more cleaning to have the parse_argument called the exact same way for every action.
I have added the parse_argument method also in the parent action object. This avoid to have it in specialized class when nothing need to be done.

Reply:
I did the cleaning and a few other things in branch  #24_46_tl